maat-court-data-api/src/main/java/gov/uk/courtdata/dto/FinAssIncomeEvidenceDTO.java

```diff
@@ -0,0 +1,30 @@
+package gov.uk.courtdata.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class FinAssIncomeEvidenceDTO implements Serializable {
+    private Integer id;
+    private LocalDateTime dateReceived;
+    private LocalDateTime dateCreated;
+    private String userCreated;
+    private LocalDateTime dateModified;
+    private String userModified;
+    private String active;
+    private LocalDateTime removedDate;
+    private String mandatory;
+    private String otherText;
+    private String adhoc;
+    private IncomeEvidenceDTO inevEvidence;
```

Comment:
IncomeEvidenceDTO is static data and should not be part of the FinAssIncomeEvidenceDTO